org.freecadweb.FreeCAD.yaml

```diff
@@ -540,10 +575,19 @@ modules:
             - -DLIBXML2_LIBRARIES=/usr/lib/aarch64-linux-gnu/libxml2.so
     build-commands:
       - sed -i 's|/usr|/app|g' ifcwrap/cmake_install.cmake
+      - sed -i 's|/usr|/app|g' ifcwrap/cmake_install.cmake
     sources:
-      - type: archive
-        url: https://github.com/IfcOpenShell/IfcOpenShell/archive/v0.6.0b0.tar.gz
-        sha256: d4a9a28b715512659670f2a101bef64363df7808a64ed30e7019161caae6a914
+      - type: git
+        url: https://github.com/IfcOpenShell/IfcOpenShell.git
+        tag: v0.7.0a2
+        commit: a11b51c971f2c3013855e13ff1929fe712ddc7de
+      - type: patch
+        path: patches/IfcOpenShell-hdf5.patch
+      - type: patch
+        options:
+          - -d
+          - src/svgfill
+        path: patches/IfcOpenShell-svgfill.patch
```

Comment:
just a not, I'm not sure if this patch is needed anymore given what the top error was. I doubt it hurts.